courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>are there meant to be TODOs and missing code in the blocks below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Ya, TODOs to change project and region. I'll add "TODO" to that line to make it abundantly clear haha.</p>